src/VirtualClient/VirtualClient.Dependencies/RedisPackageInstallation.cs

```diff
@@ -89,12 +89,15 @@ protected override async Task InitializeAsync(EventContext telemetryContext, Can
             if (this.Platform == PlatformID.Unix)
             {
                 LinuxDistributionInfo distroInfo = await this.systemManager.GetLinuxDistributionAsync(cancellationToken);
+                this.Logger.LogMessage($"Print Distro Info:{distroInfo}", telemetryContext);
```

Comment:
Please do me a favor and set the [SupportedPlatforms("linux-arm64,linux-x64", throwError: true)] and then remove the check on line 84, line 89